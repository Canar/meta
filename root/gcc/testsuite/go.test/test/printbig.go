../../../../../gcc/gcc/testsuite/go.test/test/printbig.go