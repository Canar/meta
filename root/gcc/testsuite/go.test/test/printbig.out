../../../../../gcc/gcc/testsuite/go.test/test/printbig.out